.github/workflows/macos.yml

```diff
@@ -39,7 +39,9 @@ jobs:
     - run: mkdir build
     - name: cmake
       working-directory: build
-      run: cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/${PACKAGE}/HEAD
+      run: |
+        export PYTHONPATH=$PYTHONPATH:`find /usr/local/Cellar -path '*ignition/math.cpython*' | rev | cut -d'/' -f3- | rev`
```

Comment:
Not sure why we need to use `find`. Do we not know the exact installation path?

Reply:
let's see if this works https://github.com/ignitionrobotics/sdformat/pull/957/commits/8c90c0c6248a461338951b8d69420850ef8a8501